### PR TITLE
[VEN-2712] add vip for the Liquid staked ETH pool for arbitrum sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves **#**VEN-2712**** and updated the comptroller Implementation to `0x6b9C91d7310BC19A9ce8a0AD7F926A72cEeb3b1D` as previous implementation contains the constant `MAX_COLLATERAL_FACTOR_MANTISSA` to be `0.9e18` instead it should be `0.95e18`

### CODE REVIEW

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+          Mock_weETH,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
```
Comment (chechu) [inline multisig/proposals/arbitrumsepolia/vip-015/index.ts:134]: The oracle for weETH should be a OneJumpOracle, with these properties:

Correlated token: weETH
Underlying token: ETH
Intermediate oracle: Chainlink. We will need to configure a fixed price for weETH on that oracle (i.e. 1.1)

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+          Mock_wstETH,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts]: Configured oneJumpOracle for wstETH, 9cd008c26bd5eba3b018ca161798f55d17d6dbdc
Deployment files for the oneJumpOracle: https://github.com/VenusProtocol/oracle/pull/214

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+    {
+      target: COMPTROLLER_BEACON,
+      signature: "upgradeTo(address)",
+      params: [NEW_COMPTROLLER_IMPLEMENTATION],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[Mock_wstETH, CHAINLINK_ETH_FEED, STALE_PERIOD_3M]],
+    },
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts]: Removed

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+          parseUnits("8000", 18),
+          parseUnits("800", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[Mock_weETH, CHAINLINK_ETH_FEED, STALE_PERIOD_3M]],
+    },
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts]: Removed

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts (hunk 1 above)
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts]: Configured oneJumpOracle for weETH, 9cd008c26bd5eba3b018ca161798f55d17d6dbdc
Deployment files for the oneJumpOracle: https://github.com/VenusProtocol/oracle/pull/214

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+      target: VwstETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VwstETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts:94]: Updated ProtocolSeizeShare to 1%, f48aac73de816678232376857556a7a2adcfbcca

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+      target: VweETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts:159]: Updated ProtocolSeizeShare to 1%, f48aac73de816678232376857556a7a2adcfbcca

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+      target: VWETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VWETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts:204]: Updated ProtocolSeizeShare to 1%, f48aac73de816678232376857556a7a2adcfbcca

File: multisig/simulations/arbitrumsepolia/vip-015/index.ts [hunk 8]
```diff
@@ -0,0 +1,371 @@
...
+
+          it("should have the correct price oracle", async () => {
+            expect(await comptroller.oracle()).to.equal(RESILIENT_ORACLE);
+          });
+
+          it("should have close factor = 0.5", async () => {
+            expect(await comptroller.closeFactorMantissa()).to.equal(parseUnits("0.5", 18));
+          });
+
+          it("should have liquidation incentive = 1.1", async () => {
```
Comment (chechu) [inline multisig/simulations/arbitrumsepolia/vip-015/index.ts:336]:
```suggestion
          it("should have liquidation incentive = 1.02", async () => {
```

File: multisig/simulations/arbitrumsepolia/vip-015/index.ts [hunk 9]
```diff
@@ -0,0 +1,371 @@
...
+
+          it("should have owner = GUARDIAN", async () => {
+            expect(await comptroller.owner()).to.equal(GUARDIAN);
+          });
+        });
+      };
+
+      checkComptroller(COMPTROLLER_LIQUID_STAKED_ETH, "Liquid Staked ETH");
+    });
+
```
Comment (chechu) [inline multisig/simulations/arbitrumsepolia/vip-015/index.ts:352]: Could we use `checkIsolatedPoolsComptrollers`?

Comment (Debugger022) [reply multisig/simulations/arbitrumsepolia/vip-015/index.ts:352]: `checkIsolatedPoolsComptrollers` is configured only for BNB chain

File: multisig/proposals/arbitrumsepolia/vip-015/index.ts [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumsepolia.POOL_REGISTRY, "0"],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumsepolia.POOL_REGISTRY, "78042235579982593"],
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-015/index.ts]: Updated the amount, 5fe134af7b391dd6d0a542e548fedc642dcf3124

**Review by GitGuru7 — APPROVED** [review]

**Pull request merged**